docs/docs-components/SandpackExample.js

```diff
@@ -27,43 +27,29 @@ async function copyCode({ code }: {| code: ?string |}) {
 }
 
 function SandpackContainer({
-  name,
   layout,
-  previewHeight = 'md',
+  name,
+  previewHeight,
+  showControls,
   showEditor,
-  hideControls,
-  mobileView,
 }: {|
+  layout: 'row' | 'column' | 'mobileRow',
   name: string,
-  layout: 'row' | 'column',
-  previewHeight?: 'sm' | 'md' | number,
+  previewHeight?: number,
+  showControls: boolean,
```

Comment:
It seems like this prop should be optional; however, that would imply (to me) that the default value would be `false` and we'd only add the prop when needed. But I think our desired default is to show the controls (I believe this is why Alberto initially created this as `hideControls`).

What do you think?

Reply:
@dangerismycat you both convinced me:
* changed `showControls` back to `hideControls`
* changed `showEditor` to `hideEditor` since the default is also to show the editor